apps/chat/.env.development

```
@@ -0,0 +1,16 @@
+BACKEND_URL=http://localhost:8000
+
+# Next.js API Config
+NEXT_PUBLIC_API_URL=http://localhost:8000
+
+# Backend API URL for client-side requests
+NEXT_PUBLIC_BACKEND_URL=http://localhost:8000
+
+# MCP Config
+MCP_URL=https://mcp-studies.stg.env.df-app.ch/mcp/mcp
+
+AZURE_RESOURCE_NAME=klicker-ai
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)